gce-scripts/scp_and_setup.sh

```diff
@@ -0,0 +1,139 @@
+    echo 'Creating packages log file'
+    ssh -i ~/.ssh/gce_pshtt_key ubuntu@"${machine}" touch package_log_file.txt
+    echo $?
+    # Check to see if this screen exists already.
+    ssh -i ~/.ssh/gce_pshtt_key ubuntu@"${machine}" screen -list | grep -q "package_screen"
+    answer=$(echo $?)
+    # If the screen exists, then we won't create another one. Otherwise, create.
+    if [[ "${answer}" -eq 1 ]] ; then
+      echo 'Creating screen'
+      ssh -i ~/.ssh/gce_pshtt_key ubuntu@"${machine}" screen -S package_screen -d -m
+      echo $?
+    fi
+    # Run packages_to_install and pipe to packages_log_file.txt on each machine.
+    ssh -i ~/.ssh/gce_pshtt_key -t ubuntu@"${machine}" "screen -S package_screen -X -p 0 stuff $'sudo ./packages_to_install.sh > package_log_file.txt\n'"
+    echo $?
+    ((i=i+1))
+done
+
+
+# Check that all machines have finished installing packages.
+###################################################################
+# Grab the last machine in the hosts file. This was the last one to
+# be uploaded and kicked off, so presumably it will be the last one
+# to finish.
+machine=$(sed "${num_files}q;d" $hosts_file)
+while true
+do
+    echo 'Waiting on packages to install'
+    # Wait 10 seconds before checking the file again.
+    sleep 10
+    ssh -i ~/.ssh/gce_pshtt_key ubuntu@"${machine}" tail package_log_file.txt | grep -q 'FINISHED INSTALLING PACKAGES'
+    finished=$(echo $?)
+    if [[ "${finished}" -eq 0 ]]; then
+      break
+    fi
+done
+
+# Since the last machine is finished, go check the other machines.
+i=1
+for z in $list_of_files;
+do
+    machine=$(sed "${i}q;d" $hosts_file)
+    echo 'Now on '"${machine}"' number '$i
+    echo 'Checking packages finished installing'
+    ssh -i ~/.ssh/gce_pshtt_key ubuntu@"${machine}" tail package_log_file.txt | grep -q 'FINISHED INSTALLING PACKAGES'
+    finished=$(echo $?)
+    if [[ "${finished}" -eq 0 ]]; then
+      # Check if any of the machines had a problem installing packages.
+      ssh -i ~/.ssh/gce_pshtt_key ubuntu@"${machine}" cat package_log_file.txt | grep -q '1 ERROR CODE'
+      error=$(echo $?)
+      if [[ "${error}" -eq 0 ]]; then
+        echo 'ERROR WITH '"${machine}"
+        error_with_packages=0
+      fi
+    fi
+    ((i=i+1))
+done
+
+# If any of the machines had an error with a package, stop the entire process,
+# inform the user.
+if [[ "${error_with_packages}" -eq 0 ]]; then
+    echo 'ERROR FOUND WITH PACKAGES'
+    exit 1
+fi
+
+# Upload remaining data files.
+#####################################################################
+i=1
+for y in $list_of_files;
+do
+    machine=$(sed "${i}q;d" $hosts_file)
+    echo 'Now on '"${machine}"' number '$i
+    echo 'Cloning github repo file'
+    ssh -i ~/.ssh/gce_pshtt_key -t ubuntu@"${machine}" git clone https://github.com/dhs-ncats/pshtt.git
+    echo $?
+    echo 'copying data file to pshtt directory'
+    scp -i ~/.ssh/gce_pshtt_key input_files/"${y}" ubuntu@"${machine}":~/pshtt/
```

Comment:
Similar to the comment above about an `_ssh` alias, maybe there could be a similar command for `alias _scp='scp -i ~/.ssh/gce_pshtt_key -o "StrictHostKeyChecking no" '`